main.py

```diff
@@ -0,0 +1,66 @@
+from typing import Any, Type
+
+from fastapi import FastAPI, Depends, HTTPException
+from sqlalchemy.orm import Session
+
+import crud
+from db import models
+from db.engine import SessionLocal
+from schemas import Authors, Books, CreateAuthor, CreateBook
+
+app = FastAPI()
+
+
+def get_db() -> Session:
+    db = SessionLocal()
+
+    try:
+        yield db
+    finally:
+        db.close()
+
+
+@app.get("/")
+def read_root():
+    return {"Hello": "World"}
+
+
+@app.get("/authors/", response_model=list[Authors])
+def read_authors(
+        db: Session = Depends(get_db), skip: int = 0, limit: int = 100
+) -> list[Type[models.Author]]:
```

Comment:
Not all annotations are fixed

I suppose you added this annotations with `Type`, because sqlalchemy annotates its queries results using `Type`.

But be consistent and use uniform annotations for the whole project